docs/learning/frontend/extensions/SetupAppForExtensions.md

```diff
@@ -2,8 +2,6 @@
 
 By default, every [IModelApp]($frontend) has the [ExtensionAdmin]($frontend) configured to use the Bentley Extension Service. However, additional endpoints can be configured by adding an [ExtensionLoader]($frontend) to the `ExtensionAdmin` with [ExtensionAdmin.addExtensionLoader]($frontend) or providing a different instance of the `ExtensionAdmin` to [IModelApp.startup]($frontend).
 
-There are two `ExtensionLoaders` provided, one, [ExtensionServiceExtensionLoader]($frontend), for connecting to the Bentley Extension Service, and the second, [ExternalServerExtensionLoader]($frontend), for using a different URL scheme for pulling the Extensions. The second one is most commonly used for deploying an Extension locally to debug.
```

Comment:
could you leave a note about ExternalServiceExtensionLoader if you're not removing it?

Reply:
We're going to have to re-write this whole page so I just removed the whole paragraph to be replaced later :)